src/components/PreviewDraft/PreviewDraftLightbox.jsx

```diff
@@ -143,18 +159,23 @@ class PreviewDraftLightbox extends React.Component {
     const closeButton = (
       <Button
         css={css`
-          ${closeLightboxButtonStyle} margin-top: -20px;
-          margin-right: 0;
+          ${closeLightboxButtonStyle};
           ${typeOfPreview !== 'preview' ? twoArticlesCloseButtonStyle : null};
+          margin: 0;
         `}
         stripped
         onClick={this.onClosePreview}>
         <Cross css={closeLightboxCrossStyle} />
       </Button>
     );
+
     return (
-      <div {...classes(typeOfPreview !== 'preview' ? 'two-articles' : '')}>
-        <Lightbox onClose={this.onClosePreview} closeButton={closeButton}>
+      <StyledPreviewDraft typeOfPreview={typeOfPreview}>
+        <Lightbox
+          display
+          onClose={this.onClosePreview}
+          closeButton={closeButton}
+          contentCss={lightboxContentStyle(typeOfPreview)}>
```

Comment:
Må vi ha en contentCss i tillegg til at vi har default content-styling i Lightbox OG styled component som children? Er det fks mulig flytte contentCss-innholdet til children-komponenten, evt å ta bort default stylingen i Lightbox og så må man ordne stylingen for children selv?